src/qrCard/story.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+import { storiesOf } from '@storybook/react'
+import { text, withKnobs } from '@storybook/addon-knobs'
+import Section from 'layout/section/baseSection'
+import QrCard from './index'
+
+const stories = storiesOf('Widgets|QrCard', module)
+
+stories.addDecorator(withKnobs)
+
+const style: React.CSSProperties = {
+  width: '327px',
+  padding: '16px',
+}
+
+stories.add('with title and image', () => (
+  <Section>
+    <div style={style}>
```

Comment:
why do you need this ? Can we remove it and stick to kirk widgets for layouting?